packages/p2p-v2/src/routes/AppContent/index.tsx

```diff
@@ -56,7 +56,7 @@ const AppContent = () => {
     return (
         <>
             <CloseHeader />
-            <div className='p2p-v2-tab__wrapper overflow-hidden'>
+            <div className='overflow-hidden p2p-v2-tab__wrapper'>
```

Comment:
why are we flipping this? better to have the name of the class in front and then any additional things after it

Reply:
i didn't flip it.. i have a class sorter extension for tailwind.. it got flipped on save.

Reply:
😅 can we turn that off we should have the className first, then any tailwind after